PULL_REQUEST_PROCESS.md

```diff
@@ -238,8 +238,16 @@ Read up on [Rubocop Style Guide](https://github.com/bbatsov/ruby-style-guide#com
 
 ### Today
 
+#### Automated Tests
+
 Tests are very much lacking, please feel free to check [This github issue](https://github.com/sensu-plugins/community/issues/41) for how you can help us change this.
 
+The following blog posts talk about the testing philosophy and how to actually write tests:
```

Comment:
This section makes more sense between `Today` and `Automated Tests` imo, but it's a scoping preference on my part. LGTM.